src/horust/supervisor/process_spawner.rs

```diff
@@ -173,6 +202,43 @@ fn redirect_output(
     Ok(())
 }
 
+fn open_next_chunk(base_path: &Path) -> io::Result<File> {
+    let mut count = 1;
+    let mut path = base_path.to_path_buf();
```

Comment:
I don't think you really need a pathbuf, Path implements both display and is_file. It would save an allocation

Reply:
Yup, readjusted the loop to only use the Path.